### PR TITLE
[GHA] Split VLM job by models

### PR DESCRIPTION
**Pull request by yatarkan**
CVS-156939

### CODE REVIEW

File: .github/workflows/causal_lm_cpp.yml [hunk 1]
```diff
@@ -774,31 +766,47 @@ jobs:
...
+        with:
+          ov_link: ${{ env.l_u22_ov_link }}
+      - uses: ./.github/actions/build_app
+        with:
+          build_target: 'visual_language_chat py_openvino_genai'
+      - uses: ./.github/actions/install_python_deps
       - name: Download and convert InternVL2 model
         run: |
+          # Lowering optimum and transformers version, workaround for https://huggingface.co/OpenGVLab/InternVL2-1B/discussions/7
+          python -m pip install git+https://github.com/eaidova/optimum-intel.git@ea/minicpmv
```
Comment (ilya-lavrenov) [inline .github/workflows/causal_lm_cpp.yml:803]: minicpm was merged to master

File: .github/workflows/causal_lm_cpp.yml [hunk 2]
```diff
@@ -774,31 +766,47 @@ jobs:
...
+    with:
+      model_id: llava-hf/llava-1.5-7b-hf
+      model_dir: llava_1_5_7b_ov
+      ov_link: ${{ format('{0}', env.l_u22_ov_link) }}
+
+  visual_language_chat_sample-ubuntu-llava_next:
+    uses: ./.github/workflows/job_vlm_sample_llava.yml
+    with:
+      model_id: llava-hf/llava-v1.6-mistral-7b-hf
+      model_dir: llava_v1_6_mistral_7b_ov
```
Comment (ilya-lavrenov) [inline .github/workflows/causal_lm_cpp.yml:781]: alternatively, you can use matrix like here https://github.com/openvinotoolkit/openvino.genai/blob/0acbe8788144f65c41a9cab8283b4b627454e12f/.github/workflows/causal_lm_cpp.yml#L78

File: .gitignore [hunk 3]
```diff
@@ -5,6 +5,9 @@
 !__init__.py
 !__main__.py
 
+# don't skip GitHub Actions files and directories
+!.github/**
```
Comment (Wovchena) [inline .gitignore:9]: Why would git skip `.github` folder if there were no this line?

Comment (yatarkan) [reply .gitignore:9]: Git skips `.github/build_app` directory because of `[Bb]uild*/` line in gitignore.

**Review by Wovchena — APPROVED** [review]
You can remove failing `diff` check to make the tests pass. But I propose retargeting to `master` because it seems to be fine now: https://github.com/openvinotoolkit/openvino.genai/pull/1159

**Comment by yatarkan** [comment]
> You can remove failing `diff` check to make the tests pass. But I propose retargeting to `master` because it seems to be fine now: #1159

So we do not need these changes in release branch?

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**